guides/source/autoloading_and_reloading_constants.md

```diff
@@ -84,7 +84,7 @@ Please, check the [Zeitwerk documentation](https://github.com/fxn/zeitwerk#file-
 Autoload paths
 --------------
 
-We call _autoload paths_ to the list of application directories whose contents are to be autoloaded. For example, `app/models`. Such directories represent the root namespace: `Object`.
+_Autoload paths_ is a list of application directories whose contents will be autoloaded. For example, `app/models`. Such directories represent the root namespace: `Object`.
```

Comment:
Closer to the original, and clearer that we're defining a concept:

```suggestion
We refer to the list of application directories whose contents are to be autoloaded as _autoload paths_. For example, `app/models`. Such directories represent the root namespace: `Object`.
```

I think "are to be" was better; "will" is too definite, given that constants won't be autoloaded until they're used.

Can you add `[ci skip]` to your commit message to avoid running CI for a documentation-only change? Thanks!

Reply:
Thank you for your feedback. Good point. And will also add `[ci skip]` to messages of documentation-only commits in future